docs/src/pages/production-error/ErrorDecoder.js

```diff
@@ -11,14 +13,24 @@ const ErrorMessageSection = styled('div')({
   display: 'block',
 });
 
-// use elevation={2}
-const ErrorMessageMarkdown = styled(MarkdownElement)(({ theme }) => {
-  return {
+const useStyles = makeStyles((theme) => ({
+  root: {
     boxShadow: theme.shadows['2'],
     color: theme.palette.error.main,
     padding: theme.spacing(1, 2),
-  };
-});
+  },
+}));
+
+const ErrorMessageMarkdown = (props) => {
+  const { className, ...other } = props;
+  const classes = useStyles();
+
+  return <MarkdownElement className={clsx(classes.root, className)} {...other} />;
```

Comment:
`MarkdownElement` is build with JSS, we cannot override with emotion currently, so just temporary replacing `styled()` with `makeStyles()`.